src/target/parsers/aprofile.cc

```diff
@@ -126,9 +127,8 @@ static TargetFeatures GetFeatures(TargetJSON target) {
   bool has_dotprod = (dotprod_default && !dotprod_disable) || (dotprod_support && dotprod_flag);
 
   return {
-      {"is_aarch64", Bool(is_aarch64)},
-      {"has_asimd", Bool(has_asimd)},
-      {"has_dotprod", Bool(has_dotprod)},
+      {"is_aarch64", Bool(is_aarch64)},  {"has_asimd", Bool(has_asimd)},
+      {"has_sve", Bool(has_sve)},        {"has_dotprod", Bool(has_dotprod)},
```

Comment:
Tests please 😸

Reply:
Good shout, I've added a test for this [here](https://github.com/apache/tvm/pull/14483/files#diff-66ebf57fcac747725f36ace0cce2bb5d097bc6b3d3a6c225f6a367b4fb23ff08R293).